packages/contracts/src/deploy-config.ts

```diff
@@ -473,4 +477,16 @@ export const deployConfigSpec: {
   l2TimeLockMinDelaySeconds: {
     type: 'number',
   },
+  mintManagerMintActivatedBlock: {
+    type: 'string',
+  },
+  mintManagerInitMintPerBlock: {
+    type: 'string',
+  },
+  mintManagerSlidingWindowBlocks: {
+    type: 'number',
+  },
+  mintManagerDecayingFactor: {
+    type: 'number',
+  },
```

Comment:
missing `mintManagerRecipients` and `mintManagerShares`

Reply:
These two are array types, so they cannot be added to the deploy-config.